package.json

```diff
@@ -25,42 +25,42 @@
     "start": "tsx scripts/start.ts"
   },
   "dependencies": {
-    "@azure/storage-blob": "^12.18.0",
+    "@azure/storage-blob": "^12.23.0",
     "@infoprojects/eslint-config": "^1.1.0",
     "@infoprojects/prettier-config": "^1.0.0",
     "@lit-labs/ssr": "^3.2.2",
-    "@octokit/rest": "^20.1.1",
+    "@octokit/rest": "^21.0.0",
     "@stencil-community/eslint-plugin": "^0.7.2",
     "@types/minimist": "^1.2.5",
     "@types/semver": "^7.5.8",
-    "@typescript-eslint/eslint-plugin": "^7.12.0",
-    "@typescript-eslint/parser": "^7.12.0",
+    "@typescript-eslint/eslint-plugin": "^7.16.0",
+    "@typescript-eslint/parser": "^7.16.0",
     "axe-core": "^4.9.1",
     "concurrently": "^8.2.2",
     "cpy-cli": "^5.0.0",
-    "danger": "12.3.1",
+    "danger": "12.3.3",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
     "eslint-plugin-lit": "^1.14.0",
     "eslint-plugin-only-warn": "^1.1.0",
-    "eslint-plugin-react": "^7.34.2",
+    "eslint-plugin-react": "^7.34.3",
     "minimist": "^1.2.8",
-    "nx": "19.2.0",
+    "nx": "19.4.2",
     "patch-package": "^8.0.0",
     "plop": "^4.0.1",
     "postinstall-postinstall": "^2.1.0",
-    "prettier": "^3.3.1",
-    "rimraf": "^5.0.7",
+    "prettier": "^3.3.2",
+    "rimraf": "^6.0.1",
     "semver": "^7.6.2",
     "stylelint": "^16.6.1",
-    "stylelint-config-recommended-scss": "^14.0.0",
+    "stylelint-config-recommended-scss": "^14.1.0",
```

Comment:
Deze update leidt tot de wijzigingen in de scss-files. Zie https://github.com/stylelint-scss/stylelint-config-recommended-scss/releases/tag/v14.1.0

Reply:
Check, helder